src/compute/src/logging/compute.rs

```diff
@@ -360,7 +363,8 @@ pub fn construct<A: Allocate>(
                     Datum::Uuid(peek.uuid),
                     Datum::Int64(worker as i64),
                     Datum::String(&peek.id.to_string()),
-                    Datum::Int64(peek.time as i64),
+                    // TODO: Convert to MzTimestamp.
+                    Datum::Int64(u64::from(peek.time) as i64),
```

Comment:
Another sketch `as` that we can `try_into()`.

Reply:
This will go away in the next commit where we add a real timestamp sql type.